test/MaciVoting/MaciVoting.changeCoordinatorPublicKey.t copy 2.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
```

Comment:
the file has "copy 2" in the title - is it expected?

Reply:
nope cheers - will update